### ISSUE TITLE
Research the PAL transparency handling mechanism (and document all verifiable results)

### ISSUE THREAD

**Issue by Duckwhale** [RagnarokResearchLab/ragnarokresearchlab.github.io#70]
The [original article](https://github.com/rdw-archive/RagnarokFileFormats/blob/master/PAL.MD) mentions how transparency is handled in color palettes. Unfortunately, it doesn't include any sources.

IIRC this information was provided by curiosity in one of the forums, but I don't have the link right now. 

TODOs:

- [ ] Find the original source/forum post and link it as a (hopefully credible) source
- [ ] Programmatically verify that all palettes (and SPRs, i.e., default palettes) include no alpha value
- [ ] If any do, verify that the alpha value stored in the color palette is indeed unused
- [ ] Additionally, verify that any palette using these "near-transparent" colors actually needs them to be fully transparent

Basically, some due diligence before the information is included in the documentation. Since the old repo still exists, there's no urgency.
